

class ContactAlreadyExists(Exception):
    pass


class ContactNotFound(Exception):
    pass
